cloudalerts/v2/loggers/structlog.py

```diff
@@ -11,12 +11,29 @@
 from google.cloud.logging.handlers import CloudLoggingHandler
 from google.cloud.logging.handlers.transports.background_thread import _Worker
 from pythonjsonlogger import jsonlogger
-from structlog import processors
-from structlog import stdlib
-from structlog import threadlocal
 
 from cloudalerts.v2.loggers.log_filter import censor_header
 
+dict_config = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "json": {
+            "format": "%(message)s %(lineno)d %(pathname)s %(levelname)-8s %(threadName)s",
+            "class": "pythonjsonlogger.jsonlogger.JsonFormatter",
+        }
+    },
+    "handlers": {"json": {"class": "logging.StreamHandler", "formatter": "json"}},
+    "loggers": {
+        "": {"handlers": ["json"], "level": "INFO"},
+        "werkzeug": {"level": "ERROR", "handlers": ["json"], "propagate": False},
```

Comment:
Is there where we would add other loggers that we want "white-listed" to be be removed?

Reply:
(Haven't seen a logging.config.dictConfig(dict_config) in use yet)

Reply:
We could have an exclude and include list if that is a problem that we need to solve. It isn't difficult.

Reply:
> We could have an exclude and include list if that is a problem that we need to solve. It isn't difficult.

No this was purely a question, as the followup comment was informing that I haven't seen this in practice and was just hoping to confirm that I understood how this works.